timerApp.js

```diff
@@ -78,10 +78,23 @@ TimerApp.prototype = {
     this._startNewAttempt();
   },
 
-  _setRandomBackgroundColor: function()
+  _setRandomThemeColor: function()
   {
     var themeColors = ["orange", "green", "red", "blue"];
-    this._domElement.classList.add("theme-" + TimerApp.Util.randomChoice(themeColors))
+    var randomChoice = TimerApp.Util.randomChoice(themeColors);
+    this._domElement.classList.add("theme-" + randomChoice);
+
+    document.head || (document.head = document.getElementsByTagName('head')[0]);
```

Comment:
It seems `document.head` is [pretty compatible](https://developer.mozilla.org/en-US/docs/Web/API/Document/head), but defensive coding sounds good. :-)